server/src/test/java/org/elasticsearch/search/vectors/AbstractKnnVectorQueryBuilderTestCase.java

```diff
@@ -163,16 +166,27 @@ public void testWrongFieldType() {
         KnnVectorQueryBuilder query = new KnnVectorQueryBuilder(
             AbstractBuilderTestCase.KEYWORD_FIELD_NAME,
             new float[] { 1.0f, 1.0f, 1.0f },
+            5,
             10,
             null
         );
         IllegalArgumentException e = expectThrows(IllegalArgumentException.class, () -> query.doToQuery(context));
         assertThat(e.getMessage(), containsString("[knn] queries are only supported on [dense_vector] fields"));
     }
 
+    public void testKLessThanNumCands() {
```

Comment:
Was confused a bit by the test name; shouldn't it be the other way around? :D

Reply:
@pmpailis Indeed, addressed in b5a189af78b9009f538bfc93662fa4ec8ae0e124